Affiliated-Project-Policy.md

```diff
@@ -0,0 +1,163 @@
+# OBF Affiliated Project Status Policy
+
+_Status: **Unratified Draft**. This policy is currently being developed. It has not yet been discussed beyond the Board of Directors._
+
+## Background
+
+The Open Bioinformatics Foundation (OBF) is a non-profit, volunteer-run
+group dedicated to promoting the practice and philosophy of Open Source
+software development and Open Science within the biological research community.
+
+OBF originally grew out of the volunteer projects BioPerl, BioJava
+and Biopython. Later BioRuby, BioSQL, BioLib, DAS, MOBY, EMBOSS, and
+OBDA joined the OBF umbrella, primarily as a consequence of requiring
+project hosting support, existing community relationships, and through
+informal consensus among members of OBF's Board of Directors. OBF
+has since become hesitant to include new projects under the official
+umbrella, in part because the growth in OBF-hosted project resources
+has reached the capacity of OBF servers, and in part because of
+OBF's mission and scope as an organization. Furthermore, the OBF Board resolved that the [listing of projects related to OBF](http://www.open-bio.org/wiki/Projects)
```

Comment:
suggest we don't link to the deprecate site - https://www.open-bio.org/projects/ would be better.

Reply:
Fixed in 8f483366b916fb9ea63d2268036299f815bf140d